docs/api/sim-card-usage.md

```diff
@@ -17,25 +19,27 @@ When querying it is also required to specify which SIM card the usage should be
 This is done using the **ICC** identifier of a SIM card.
 
 ## Data usage for a SIM card
+
 Data usage for a given **month** in a given **year** can be accessed by using the following URL:
 
 `GET /simcards/{icc}/usage/data/{year}/{month}`
 
 **Response structure and types**
 
 | Field       | Type              | Description                                                   |
-| ----------- | ----------------- | ------------------------------------------------------------- |
+|-------------|-------------------|---------------------------------------------------------------|
 | icc         | String            | The unique identifier for the SIM card generating the usage.  |
-| year        | Number            | Year is given in the ISO 8601 format for example 2019.             |
+| year        | Number            | Year is given in the ISO 8601 format for example 2019.        |
 | month       | Number            | Month is represented as a number for example 'april = 4'      |
 | usageDate   | ISO 8601 DateTime | In UTC time.                                                  |
 | bytesIn     | Number            |                                                               |
 | bytesOut    | Number            |                                                               |
 | sessionId   | String            | Unique identifier for the session on the network.             |
 | imei        | String            | The observed IMEI number for the device generating the usage. |
```

Comment:
Suggestion to add a `<br>` in description text, to create multiline text. With current text length I get a scroll bar at the bottom.